public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -804,6 +810,13 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> impleme
     return Boolean(this.getManagerKind() === ManagerKind.Repo);
   }
 
+  managerAllowsEdits() {
```

Comment:
This method checks for managed resources which cannot be edited, so I think the name needs to be updated to match that.